### PR TITLE
IfcDistributionSystem Rail Relocated Pset_EnvironmentalCondition

### PR DESCRIPTION
**Pull request by jmirtsch**
Pset_EnvironmentalCondition isolated into specific pull request

### CODE REVIEW

File: IFC4x3/Properties/a/AllowedStorageTemperature_0T6ELr2bv1MP48HEd_Ue3i/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Maximum storage temperature that system complies with (C°)
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AllowedStorageTemperature_0T6ELr2bv1MP48HEd_Ue3i/Documentation.md]:
```suggestion
Maximum storage temperature that the system complies with.
```

File: IFC4x3/Properties/a/AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Maximum working temperature that system complies with (C°)
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW/Documentation.md]:
```suggestion
Maximum working temperature that the system complies with.
```

File: IFC4x3/Properties/a/AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW" Name="AllowedWorkingTemperature" UniqueId="ef6427c2-4258-457b-ba93-3aba4523c3a0" PropertyType="p_boundedvalue" PrimaryDataType="IfcThermodynamicTemperatureMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW/DocProperty.xml]: How about `OperationalTemperatureRange`? Could modify the documentation for it.

File: IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AtmosphericPressure_0Qno0T109Cww7rcstWuI3" Name="AtmosphericPressure" UniqueId="1ac7201d-0402-4ceb-a1f5-fe6db7838483" PrimaryDataType="IfcPressureMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/DocProperty.xml]: Reuse or rename to something like Maximum... or Nominal...?

File: IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Maximum level of solar radiation that the equipment complies with, usually measured by duration of solar radiation.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md]: Duration seems like a time measure. Would that be better than `IfcReal`?

File: IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn" Name="MaximumWindSpeed" UniqueId="65dcb119-2c67-44cd-a9b3-f1b6e4d74571" PrimaryDataType="IfcLinearVelocityMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml:2]: Is this different than `WindLoadRating`?

File: IFC4x3/Properties/m/MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX/Documentation.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Minimum working temperature that system complies with (C°)
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX/Documentation.md]:
```suggestion
Minimum working temperature that the system complies with.
```

File: IFC4x3/Properties/m/MinimumStorageTemperature_0EEQzV6qrBOw1KDdYos7$D/Documentation.md [hunk 8]
```diff
@@ -0,0 +1 @@
+Minimum storage temperature that system complies with (C°)
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MinimumStorageTemperature_0EEQzV6qrBOw1KDdYos7$D/Documentation.md]:
```suggestion
Minimum storage temperature that the system or equipment can operate effectively at.
```

File: IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/Documentation.md [hunk 9]
```diff
@@ -0,0 +1 @@
+Maximum level of rain intensity that the equipment complies with
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/Documentation.md:1]:
```suggestion
Maximum level of rain intensity that the equipment complies with.
```

File: IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/DocProperty.xml [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RainIntensity_1HJs_aZFnBKP1bnlrIzkxt" Name="RainIntensity" UniqueId="514f6fa4-8cfc-4b51-9065-c6fd52f6eef7" PrimaryDataType="IfcLinearVelocityMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/DocProperty.xml]: Velocity measure? Unexpected. 😄

File: IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/Documentation.md [hunk 11]
```diff
@@ -0,0 +1 @@
+Maximum level of salt mist that the equipment complies with
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/Documentation.md:1]:
```suggestion
Maximum level of salt mist that the equipment complies with.
```

File: IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md [hunk 12]
```diff
@@ -0,0 +1 @@
+Maximum level of earthquake that the equipment complies with
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md]: Would probably make sense to be a bit more specific. Which scales can be used for example. "level of earthquake" is very ambiguous.
```suggestion
Maximum level of earthquake that the equipment complies with.
```

File: IFC4x3/Properties/s/Smoke_24x_PWjRXByRIpwhg2SIFm/Documentation.md [hunk 13]
```diff
@@ -0,0 +1 @@
+Maximum level of smoke that the equipment complies with
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/Smoke_24x_PWjRXByRIpwhg2SIFm/Documentation.md]: Same as above "level of smoke" is ambiguous.
```suggestion
Maximum level of smoke that the equipment complies with.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_EnvironmentalCondition/DocPropertySet.xml [hunk 14]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_EnvironmentalCondition/DocPropertySet.xml:1]: We need a description.

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Everything is in the comments.

File: IFC4x3/Properties/a/AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW/DocProperty.xml (hunk 3 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/a/AllowedWorkingTemperature_3lP2V2GbX5UxgJEhf58yEW/DocProperty.xml]: Seconded - `OperationalTemperatureRange_2hvjQhIJj5c9PrmgrOQl1i`

File: IFC4x3/Properties/a/AllowedStorageTemperature_0T6ELr2bv1MP48HEd_Ue3i/DocProperty.xml [hunk 15]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AllowedStorageTemperature_0T6ELr2bv1MP48HEd_Ue3i" Name="AllowedStorageTemperature" UniqueId="1d18e575-0a5e-4159-9108-44e9fe7a80ec" PropertyType="p_boundedvalue" PrimaryDataType="IfcThermodynamicTemperatureMeasure" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/a/AllowedStorageTemperature_0T6ELr2bv1MP48HEd_Ue3i/DocProperty.xml]: this says allowed but dscription is Maximum, suggest using MaximumStorageTemperature

File: IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/DocProperty.xml (hunk 4 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/DocProperty.xml]: currently AtmosphericPressure as a property is a performance timeseries property so a different name is required maybe  `MaximumAtmosphericPressure` or do you need an operational range (depends on how the property is reported usually

File: IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/Documentation.md [hunk 16]
```diff
@@ -0,0 +1 @@
+Maximum level of atmospheric pressure that the equipment complies with
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/Documentation.md]:
```suggestion
Maximum level of atmospheric pressure that the equipment can operate effectively in.
```

File: IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md (hunk 5 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md]: as with sergej is this an intensity/density measurement therfore solar irradiance is measured in W/m2 which is a 'Power density' that does not exist yet, if it is inintegrated over time then becomes J/m2 which is an Energy density again isnt a measure for that.... i would add an example SI based unit like above to the description so people know what it should be until we have the measures added to #336

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_EnvironmentalCondition/DocPropertySet.xml [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_EnvironmentalCondition" UniqueId="f70c35a7-7534-a696-f015-8118733a96ab" ApplicableType="IfcFurniture/TECHNICALCABINET,IfcSwitchingDevice/SELECTORSWITCH,IfcElementAssembly/TRACTION_SWITCHING_ASSEMBLY,IfcSwitchingDevice/EMERGENCYSTOP,IfcProtectiveDevice/FUSEDISCONNECTOR,IfcElectricFlowTreatmentDevice/ELECTRONICFILTER,IfcActuator/ELECTRICACTUATOR,IfcElectricFlowStorageDevice/COMPENSATOR,IfcElectricFlowStorageDevice/RECHARGER,IfcDistributionElement" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_EnvironmentalCondition/DocPropertySet.xml]: Do we need such a specific list for this or can we just apply it to IfcElement like Pset_EnvironmentalImpactIndicators?

File: IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml (hunk 6 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml:2]: wind load rating is usually a national class structure based on tests so is an IfcLabel, how is this condition limit measured and reported for railways?

File: IFC4x3/Properties/m/MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX/DocProperty.xml [hunk 18]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX" Name="MinimumOperatingTemperature" UniqueId="4d1abcb4-b0f6-481e-87d5-d30cb5e59361" PrimaryDataType="IfcThermodynamicTemperatureMeasure" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/m/MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX/DocProperty.xml]: if you use `OperationalTemperatureRange_2hvjQhIJj5c9PrmgrOQl1i` above as suggested you do not need this as the lower bound of that property is this value.

File: IFC4x3/Properties/m/MinimumStorageTemperature_0EEQzV6qrBOw1KDdYos7$D/Documentation.md (hunk 8 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/m/MinimumStorageTemperature_0EEQzV6qrBOw1KDdYos7$D/Documentation.md]: or maybe a bounded value for `StorageTemperatureRange` with:

> Effective storage temperature range that the system or equipment can operate effectively at.

File: IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/DocProperty.xml (hunk 10 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/DocProperty.xml]: well unit wise technically yes, cause it's the amount of rain that falls (measured in millimetres usually) over time so length/duration. which is technically velocity. but @czapplitec dont forget the semantic content of the measure which is not correct for this case. 

Suggestion: use IfcReal with typical unit details in the description. And i have added it to #336

File: IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/Documentation.md (hunk 9 above)
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/Documentation.md]:
```suggestion
Maximum level of rain intensity that the equipment can operate effectively in.
```

File: IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/Documentation.md (hunk 11 above)
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/Documentation.md]:
```suggestion
Maximum level of salt mist that the equipment can operate effectively in.
```

File: IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/DocProperty.xml [hunk 19]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SaltMist_3rzK_24B55FhlIh99zlW26" Name="SaltMist" UniqueId="f5f54f82-10b1-453e-bbd2-ac927dbe0086" PrimaryDataType="IfcReal" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/DocProperty.xml]: would appreciate knowning that the measurement unit for this described as?

File: IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md (hunk 12 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md]: same as sergej, if you know the measurement type we can either select the appropriate one or add it to the list in #336

File: IFC4x3/Properties/s/Smoke_24x_PWjRXByRIpwhg2SIFm/Documentation.md (hunk 13 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/s/Smoke_24x_PWjRXByRIpwhg2SIFm/Documentation.md]: can we have the measurement unit please!

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
some clarifcations on units and such and documentation

File: IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md [hunk 20]
```diff
@@ -1 +1 @@
-Maximum level of solar radiation that the equipment complies with, usually measured by duration of solar radiation.
+Maximum level of solar radiation that the equipment complies with. The value indicates power density measured in watt per square meter (w/m2).
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/m/MaximumSolarRadiation_3GwIZXL0151QO$6Rwbh8Tj/Documentation.md]:
```suggestion
Maximum level of solar irradiance that the equipment can operate effectively in. This is usually tested and measured by a national or international standard. The value indicates power density measured in watt per square meter (w/m2).
```
suggest name change to `MaximumSolarIrradiance` which is the correct scientific naming for the intended process being described.

File: IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md [hunk 21]
```diff
@@ -1 +1 @@
-Maximum level of earthquake that the equipment complies with
+Maximum level of earthquake that the equipment complies with.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md]: any ideas on measurement type?

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
just a suggestion on naming and just clarify a measurement unit then this is all done!

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM, can we just have an answer to my query on MaximumWindSpeed

> wind load rating is usually a national class structure based on tests so is an IfcLabel, how is this condition limit measured and reported for railways?

File: IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml (hunk 6 above)
Comment (czapplitec) [reply IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml:2]: The requirement is the wind speed, measured in m/s.

Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml:2]: thank you! thats all i wanted to know!

**Review by AlexBrad1eyCT — APPROVED** [review]
All good, just @SergejMuhic to provde second approval!

File: IFC4x3/Properties/s/SaltMistLevel_3rzK_24B55FhlIh99zlW26/Documentation.md [hunk 22]
```diff
@@ -0,0 +1 @@
+Maximum level of salt mist that the equipment can operate effectively in. It is provided according to international or national standard.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SaltMistLevel_3rzK_24B55FhlIh99zlW26/Documentation.md]:
```suggestion
Maximum level of salt mist that the equipment can operate effectively in. It is provided according to an international or national standard.
```

File: IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md [hunk 23]
```diff
@@ -0,0 +1 @@
+Maximum level of earthquake that the equipment complies with. The value indicates earthquake intensity measured in Richter scale.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md]: Mximum magnitude or scale seems better wording. Let us resolve this together in the meeting.

File: IFC4x3/Properties/s/SmokeLevel_24x_PWjRXByRIpwhg2SIFm/Documentation.md [hunk 24]
```diff
@@ -0,0 +1 @@
+Maximum level of smoke that the equipment complies with. It is provided according to international or national standard.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/SmokeLevel_24x_PWjRXByRIpwhg2SIFm/Documentation.md]:
```suggestion
Maximum level of smoke that the equipment complies with. It is provided according to an international or national standard.
```

File: IFC4x3/Properties/s/StorageTemperatureRange_0T6ELr2bv1MP48HEd_Ue3i/Documentation.md [hunk 25]
```diff
@@ -0,0 +1 @@
+Allowed storage temperature that the element complies with.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/StorageTemperatureRange_0T6ELr2bv1MP48HEd_Ue3i/Documentation.md]:
```suggestion
Allowed storage temperature range that the element complies with.
```

**Review by SergejMuhic — COMMENTED** [review]
Can talk about this one first. Close to merging.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**